webpack.config.test.js

```diff
@@ -0,0 +1,8 @@
+const nodeExternals = require( 'webpack-node-externals' );
+
+const baseConfig = require( './webpack.config' );
+
+const config = module.exports = Object.assign( {}, baseConfig, {
+	target: 'node',
+	externals: [ nodeExternals() ],
```

Comment:
I tried without it and using your alternative. I don't understand all what's happening here but it's not working without this.